rs/ic_os/config/src/deployment_json.rs

```diff
@@ -322,26 +322,27 @@ mod test {
 
     #[test]
     fn deserialize_deployment() {
-        let parsed_deployment: DeploymentJson = { serde_json::from_str(DEPLOYMENT_STR).unwrap() };
+        let parsed_deployment: DeploymentSettings =
+            { serde_json::from_str(DEPLOYMENT_STR).unwrap() };
```

Comment:
The {} seem unnecessary

Reply:
Fixed in https://github.com/dfinity/ic/pull/1539